.github/workflows/ci.yml

```diff
@@ -27,15 +27,15 @@ jobs:
         julia-arch:
           - x64
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - uses: julia-actions/setup-julia@v1
+      - uses: julia-actions/setup-julia@v2
         with:
           version: ${{ matrix.julia-version }}
           arch: ${{ matrix.julia-arch }}
       - name: Cache artifacts
-        uses: actions/cache@v2
+        uses: actions/cache@v4
```

Comment:
Use `julia-actions/cache` instead?

Reply:
Done in https://github.com/JuliaEcosystem/PackageAnalyzer.jl/pull/116/commits/945c78d703c1722ff7e22403a30f01cfec3a2226 and https://github.com/JuliaEcosystem/PackageAnalyzer.jl/pull/116/commits/35cd6c307fdf4a951e89caf754b0e66e2725ea87.